pkg/controller/istiocontrolplane/istiocontrolplane_controller.go

```diff
@@ -140,9 +144,9 @@ func (r *ReconcileIstioControlPlane) Reconcile(request reconcile.Request) (recon
 		finalizers = append(finalizers[:finalizerIndex], finalizers[finalizerIndex+1:]...)
 		instance.SetFinalizers(finalizers)
 		finalizerError := r.client.Update(context.TODO(), instance)
-		for retryCount := 0; errors.IsConflict(finalizerError) && retryCount < 5; retryCount++ {
+		for retryCount := 0; errors.IsConflict(finalizerError) && retryCount < finalizerMaxRetries; retryCount++ {
 			// workaround for https://github.com/kubernetes/kubernetes/issues/73098 for k8s < 1.14
-			reqLogger.Info("confilict during finalizer removal, retrying")
+			reqLogger.Info("conflict during finalizer removal, retrying")
```

Comment:
a human operator will not have the faintest clue what this info means.  I dont htink a log here is really necessary, unless this is just for debug.  Might be better to lower the level of this error if the string is kept as is.

Reply:
or reword is to be more meaningful, like Error during istio control plane removal, retrying